frontend/src/app/page.tsx

```diff
@@ -1,21 +1,29 @@
-import React from "react";
-import Logo from "./components/HomeComponents/Logo/logo";
-import "tailwindcss/tailwind.css";
-import IntroSignPost from "./components/HomeComponents/introsignpost";
-import LandingImage from "./components/LandingImage";
+import React from 'react';
+import { type Metadata } from 'next';
 
```

Comment:
Again, blank row?

Reply:
I prefer to leave a free line between logical units for a better structure of imports, if you don't mind :)

Reply:
Alright, let's have it there. I won't be that d*ck next time :)